packages/flutter/lib/src/material/tabs.dart

```diff
@@ -104,16 +112,16 @@ class Tab extends StatelessWidget implements PreferredSizeWidget{
   Widget build(BuildContext context) {
     assert(debugCheckHasMaterial(context));
 
-    final double height;
+    final double tabHeight;
```

Comment:
To remove some redundant code here, you could leave it as is here with a new name
```suggestion
    final double calculatedHeight;
```